### DISCUSSION TITLE
Minimum fee required - Dust Storm

### DISCUSSION THREAD

**Discussion by majekqwert** [Chia-Network/chia-blockchain#17172] [Ideas]
I think you need to add the minimum required fee, e.g. 1 mojo. To prevent clogging up the meempool. Currently, the network is functioning terribly

**Comment by BrandtH22** [comment]
Hey @majekqwert , please note that any farmer is able to update their nodes code to require a minimum fee but when we attempting launching a min fee for the network the community shot down the idea (https://github.com/Chia-Network/chips/pull/13). 

This being said chia does have a minimum effective fee of 5 mojo per clvm cost. This means that anyone who is trying to submit fees with their transaction must use the minimum effective fee for it to be recognized (anything less is no different than a 1 mojo fee). We have taken the time to provide estimates for this minimum effective fee here https://docs.chia.net/coin-set-costs/#estimated-transaction-costs.

Since this idea had already been worked through the CHIP process and was rejected by the community, I will be closing this discussion, but if you believe it was closed in error feel free to reopen it to continue the discussion or join us on discord (https://discord.gg/chia).

**Comment by majekqwert** [comment]
The issue is that currently someone is spamming the network by sending numerous transactions without fees, making it necessary to apply a fee of 0.01 XCH for a regular transaction to go through. The discussion on this matter took place over a year ago, and since then, many things have changed. Therefore, it would be worthwhile to reconsider the proposal of adding a network-level fee.
